docs/en/setup/service-agent/java-agent/Supported-list.md

```diff
@@ -80,6 +80,8 @@
   * Vert.x Web 3.x
 * Thread Schedule Framework
   * [Spring @Async](https://github.com/spring-projects/spring-framework) 4.x and 5.x
+* Cache
+  * [Ehcache](https://www.ehcache.org/) 2.x
```

Comment:
@wu-sheng do we want to put this to section `* NoSQL`? I see some cache frameworks like Redis and Memcached are both there

Reply:
I think we could just put ehcache as level one. Ehcache is just a in memory cache and local span only.